src/mcts/node.h

```diff
@@ -240,6 +243,22 @@ class Node {
   // Debug information about the node.
   std::string DebugString() const;
 
+  // Reallocates this nodes children to be in a solid block, if possible and not already done.
+  // Returns true if the transformation was performed.
+  bool MakeSolid();
```

Comment:
The comment at the top should be updated too:
https://github.com/LeelaChessZero/lc0/blob/b4f1dbc60a2befe4e56384441ac4a91e2972354b/src/mcts/node.h#L51-L52

Bonus points for the diagram too! 😛

Reply:
I think maybe I'll give the diagram a miss, but updated the text :)